app/components/post_draft/quick_actions/file_quick_action/index.tsx

```diff
@@ -3,25 +3,33 @@
 
 import React, {useCallback} from 'react';
 import {useIntl} from 'react-intl';
-import {Alert, StyleSheet} from 'react-native';
+import {Alert, Text} from 'react-native';
 
 import CompassIcon from '@components/compass_icon';
 import TouchableWithFeedback from '@components/touchable_with_feedback';
 import {ICON_SIZE} from '@constants/post_draft';
 import {useTheme} from '@context/theme';
 import {fileMaxWarning} from '@utils/file';
 import PickerUtil from '@utils/file/file_picker';
-import {changeOpacity} from '@utils/theme';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
 
 import type {QuickActionAttachmentProps} from '@typings/components/post_draft_quick_action';
 
-const style = StyleSheet.create({
-    icon: {
+const getStyle = makeStyleSheetFromTheme((theme: Theme) => ({
+    title: {
+        color: theme.centerChannelColor,
+        fontSize: 16,
+        fontWeight: 400,
+        lineHeight: 24,
```

Comment:
Instead of adding the fontSize, etc... directly, use the typography util function. Check all other files for the same.

Reply:
Done.